src/app/desktop/components/Instances/Instance.js

```diff
@@ -195,7 +195,7 @@ const Instance = ({ instanceName }) => {
     } else {
       setBackground(`${instanceDefaultBackground}`);
     }
-  }, [instance.background, instancesPath, instanceName]);
+  }, [instance, instancesPath, instanceName]);
```

Comment:
This is really bad from a performance perspective. Is there a reason why we need to trigger this on every instance change?